swagger2locustio/templates/locustfile_templates.py

```diff
@@ -23,7 +22,7 @@ def on_start(self):
 class WebsiteUser(HttpLocust):
     task_set = Tests
     wait_time = between(5.0, 9.0)
-    {% if host %}
+{% if host %}
```

Comment:
tabulation is very important in the case of generation locustfile. have you checked file generation after changing this line?

Reply:
yes, I've checked it